### PR TITLE
Cache home page

### PR DESCRIPTION
**Pull request by sharph**

### CODE REVIEW

**Comment by sharph** [comment]
@hectcastro, feel free to have a look too.

**Comment by ddohler** [comment]
Looks fine to me, not something I know a lot about though.

File: deployment/ansible/roles/ckan-odp-configuration/templates/nginx_ckan.j2 [hunk 1]
```diff
@@ -23,4 +23,16 @@ server {
         # In emergency comment out line to force caching
         # proxy_ignore_headers X-Accel-Expires Expires Cache-Control;
     }
+    location = / {
+        proxy_pass http://127.0.0.1:8080/;
+        proxy_set_header Host $host;
+        proxy_cache cache;
+        proxy_cache_bypass $cookie_auth_tkt;
```
Comment (hectcastro) [inline deployment/ansible/roles/ckan-odp-configuration/templates/nginx_ckan.j2]: Guessing that this prevents Nginx from caching if the user is logged in, but having trouble figuring out what `$cookie_auth_tkt` actually means?

Comment (sharph) [inline deployment/ansible/roles/ckan-odp-configuration/templates/nginx_ckan.j2]: If the cookie auth_tkt exists...

Comment (sharph) [inline deployment/ansible/roles/ckan-odp-configuration/templates/nginx_ckan.j2]: This was actually in the default CKAN config, so there wasn't much to figure out.

**Comment by hectcastro** [comment]
Seems like a safe change. Looks pretty much identical to the default.

:+1:

**Pull request merged**